.github/workflows/tests_on_request.yml

```diff
@@ -8,13 +8,13 @@ jobs:
   CI_commit:
     if: >-
       github.event.issue.pull_request &&
-      contains(github.event.comment.body, 'Please test this!')
```

Comment:
Can you make this case insensitive, and just check if it has both @PyKEEN-bot and "test"? Might reduce the number of mistakes/extra comments

Reply:
The contains function is case insensitive by default. I agree with the latter one.